docs/ansible/ansible.md

```diff
@@ -7,6 +7,8 @@ Depending on your available python version you may be limited in choosing which
 
 It is recommended to deploy the ansible version used by kubespray into a python virtual environment.
 
+Download kubespray git directory and run below commands at the parent directory of`kubespray`:
```

Comment:
This does not really belong there, this file is about who to install Ansible.

Reply:
But below content is about KubeSpray, and if not existed, it would fail.

Reply:
You didn't modify the code block, why did it fail?

Reply:
It needs something from KubeSpray.

Reply:
That's not not exactly the point. Yes, this instruction assume you already have cloned kubespray (or have a copy, extract an archive, etc). This file is about installing ansible, and the docs assume you already have a copy of kubespray. If we didn't do that, we'd have to copy that sentence everywhere in the docs.

In other words, this is already part of the context here.

Reply:
You are right, that is the problem. 

As usual, third-party tool should be installed without any dependence, but now, it is close to KubeSpray. So it worth comment.

Reply:
I don't understood your last message ; however, my previous explain why this is in fact not a good place for this comment.